docs/install-manual.md

```diff
@@ -0,0 +1,144 @@
+
+This example uses the Docker Hub based `token` discovery service. Log into **each node** and do the following.
+
+1. Start the Docker daemon with the `-H` flag. This ensures that the Docker remote API on *Swarm Agents* is available over TCP for the *Swarm Manager*.
+
+    		$ docker -H tcp://0.0.0.0:2375 -d
+ 
+
+2. Register the Swarm agents to the discovery service. The node's IP must be accessible from the Swarm Manager. Use the following command and replace with the proper `node_ip` and `cluster_id` to start an agent:
+
+		```bash
+		docker run -d swarm join --addr=<node_ip:2375> token://<cluster_id>
+		```
+
+  For example:
+
+		$ docker run -d swarm join --addr=172.31.40.100:2375 token://6856663cdefdec325839a4b7e1de38e8
+
+3. Start the Swarm manager on any machine or your laptop. 
+
+	The following command illustrates how to do this:
+
+	```bash
+	docker run -d -p <swarm_port>:2375 swarm manage token://<cluster_id>
+	```
+
+4. Once the manager is running, check your configuration by running `docker info` as follows:
+
+	```bash
+	docker -H tcp://<manager_ip:manager_port> info
+	```
+
+	For example, if you run the manager locally on your machine:
+
+      $ docker -H tcp://0.0.0.0:2375 info
+      Containers: 0
+      Nodes: 3
+       agent-2: 172.31.40.102:2375
+        └ Containers: 0
+        └ Reserved CPUs: 0 / 1
+        └ Reserved Memory: 0 B / 514.5 MiB
+       agent-1: 172.31.40.101:2375
+        └ Containers: 0
+        └ Reserved CPUs: 0 / 1
+        └ Reserved Memory: 0 B / 514.5 MiB
+       agent-0: 172.31.40.100:2375
+        └ Containers: 0
+        └ Reserved CPUs: 0 / 1
+        └ Reserved Memory: 0 B / 514.5 MiB
+
+  If you are running a test cluster without TLS enabled, you may get an error.
+  In that case, be sure to unset `DOCKER_TLS_VERIFY` with:
+
+    $ unset DOCKER_TLS_VERIFY
+  
+## Using the docker CLI
+
+You can now use the regular Docker CLI to access your nodes:
+
+```bash
+docker -H tcp://<manager_ip:manager_port> info
+docker -H tcp://<manager_ip:manager_port> run ...
+docker -H tcp://<manager_ip:manager_port> ps
+docker -H tcp://<manager_ip:manager_port> logs ...
+```
+
+## List nodes in your cluster
+
+You can get a list of all your running nodes using the `swarm list` command:
+
+```bash
+docker run --rm swarm list token://<cluster_id>
+<node_ip:2375>
+```
+
+For example:
+
+```bash
+$ docker run --rm swarm list token://6856663cdefdec325839a4b7e1de38e8
+172.31.40.100:2375
+172.31.40.101:2375
+172.31.40.102:2375
+```
+## TLS
+
+Swarm supports TLS authentication between the CLI and Swarm but also between
+Swarm and the Docker nodes. _However_, all the Docker daemon certificates and client
+certificates **must** be signed using the same CA-certificate.
+
+In order to enable TLS for both client and server, the same command line options
+as Docker can be specified:
+
+```bash
+swarm manage --tlsverify --tlscacert=<CACERT> --tlscert=<CERT> --tlskey=<KEY> [...]
```

Comment:
I arrived here to check if the rewrite resolves https://github.com/docker/swarm/issues/722

Should this also be a  `docker run ..... swarm manage ...`? Or will that become too complicated here with passing the certificates?